docker-compose.yml

```diff
@@ -38,4 +43,4 @@ services:
     volumes:
       - .:/app
```

Comment:
In general this looks fine! I have one question, does Sidekiq absolutely need to mount the current directory as a volume? We're trying to avoid this in the first place as described in #2776.

Reply:
This looks fine but I didn't test it - it'd be ideal if you could test it locally with `docker-compose` or else we could test it in `unstable`.

Reply:
We could test it on unstable!! Will do it.